src/orange/OrangeTrackView.hh

```diff
@@ -243,6 +246,15 @@ CELER_FUNCTION real_type OrangeTrackView::find_next_step()
     return next_step_;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Find the nearest (any direction) boundary within the current volume.
+ */
+CELER_FUNCTION real_type OrangeTrackView::find_safety(const Real3&)
+{
+    CELER_NOT_IMPLEMENTED("safety distance in ORANGE");
+}
```

Comment:
should return auto for now?

Reply:
Aha, looks like the CUDA compiler isn't seeing that the function cannot ever return -- strange because it should have marked it as an "unreachable" code point on that version of CUDA. I'll just `return {}` so the interface is correct and the warning goes away.